narratives/ncov_sit-rep_2020-05-01.md

```diff
@@ -0,0 +1,259 @@
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Five local transmission chains in the Czech Republic](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Czech%20Republic&m=div&p=full)
+
+The data from the Czech Republic show a mix of these two patterns.
+<br><br>
+There are at least 7 instances where a sample isolated in the Czech Republic descends from a virus that was mostly likely circulating elsewhere in Europe, indicating new introductions between March 1 and April 2.
+<br><br>
+For 5 of these introductions, we also see closely related local cases, sampled over time. This is consistent with local transmission.
+<br><br>
+From this, we can say that there were at least 7 introductions to the Czech Republic; at least 5 of these likely led to further local transmission.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Extensive mixing of cases across Turkey](https://nextstrain.org/ncov/europe/2020-04-29?c=country_exposure&d=tree,map&f_country=Turkey&label=clade:A2&p=full&r=division&legend=open)
+
+In the middle of the tree, we see a string of cases sampled from many different provinces around Turkey.
+Some of these cases have reported travel history to Saudi Arabia, with subsequent closely related cases in Turkey.
+<br><br>
+Normally, we might assume that returning travelers had passed the virus to household members or neighbors back home.
+Interestingly, though, the travelers' cases were often sampled from different provinces than the most closely related cases.
+<br><br>
+This indicates that there has been extensive mixing of viral lineages, reaching many provinces of Turkey within one week.
+We may also speculate that a large gathering could explain these observations, although we would need additional epidemiological data to further investigate.
+<!-- There is no right-side text -->
+
+
+<!-- There is no right-side text -->
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text -->
+# [At least 11 introductions to Poland](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Poland&p=full)
+
+We have 15 sequences from Poland on Nextstrain. Of these 15, at least 11 of them represent separate introductions from elsewhere in the world.
+As in Moscow, given the number of reported cases, we might expect that several of these introductions have seeded local transmission chains in Poland, but we can't yet see this in the genomic data.
+<!-- There is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Early samples from Hungary, Latvia, Slovakia and Slovenia](https://nextstrain.org/ncov/europe/2020-04-29?d=tree,map&f_country=Hungary,Latvia,Slovakia,Slovenia&f_region=Europe&p=full)
```

Comment:
Would remove section, since we don't really say anything about transmission

Reply:
@nicfel @cassiawag -- I'm a little torn here. I threw this in just to acknowledge that we have data from these areas and think it's important, even though we can't yet say much about it. Votes on whether to keep it in or remove it?

Reply:
I'm ok with it myself, I think it's fair to recognise the effort. But no strong feelings.